step-functions-data-science-sdk/automate_model_retraining_workflow/automate_model_retraining_workflow.ipynb

```diff
@@ -133,7 +133,7 @@
     "2. Select **Roles** and then **Create role**.\n",
```

Comment:
<p>Suggestion: attach the policy automatically via boto3. i..e automatically fill the NOTEBOOK_ROLE_ARN from the previous step; put the entire policy in a dictionary; then use certain boto3 api to attach it to the role.</p><p> </p><p>It's easier for user to read/understand the content of the notebook if they don't have to navigate to the console and do all those steps via GUI</p><p> </p><p>It is also easier for automated testing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1960/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>